<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevaResolucion</class>
 <widget class="QDialog" name="dlgNuevaResolucion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtEpigrafe">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Epígrafe</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>691</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tabResolucion">
    <attribute name="title">
     <string>Resolución</string>
    </attribute>
    <widget class="QTextEdit" name="txtResolucion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>561</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtResolucionTraduccion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>641</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabResumen">
    <attribute name="title">
     <string>Resumen</string>
    </attribute>
    <widget class="QTextEdit" name="txtResolucionResumen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>531</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabNotas">
    <attribute name="title">
     <string>Notas</string>
    </attribute>
    <widget class="QTextEdit" name="txtResolucionNotas">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>581</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabTemas">
    <attribute name="title">
     <string>Temas</string>
    </attribute>
    <widget class="QComboBox" name="cboTemas">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btAnadirTema">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Añadir tema</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btQuitarTema">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrar tema</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btNuevoTema">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>220</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Nuevo tema</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="twTemas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <column/>
    </widget>
   </widget>
   <widget class="QWidget" name="tabDetalles">
    <attribute name="title">
     <string>Detalles</string>
    </attribute>
    <widget class="QTreeWidget" name="treeDetalles">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="cboDescripcion">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Descripción</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboLugares">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Lugares</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btJsonAnadirDescripcion">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btJsonAnadirLugar">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>70</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btNuevoJson">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Nuevos datos</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboPersonas">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cboCasas">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>150</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>120</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Personas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Casas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btJsonAnadirPersona">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>110</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btJsonAnadirCasa">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>150</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtKey">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtValue">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btJsonAnadirLibre">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>200</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>200</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Libre</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btNuevoLugar">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btNuevaPersona">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>110</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btNuevaCasa">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>150</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btBorrarElemento">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrar elemento</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btModificarDetalles">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Añadiendo</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabOtros">
    <attribute name="title">
     <string>Otros</string>
    </attribute>
    <widget class="QCheckBox" name="chEntendida">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Entendida</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chVolverMirar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Volver a mirar</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chTradudida">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Traducida</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spInteresante">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interesante</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btOK">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
